### PR TITLE
Make it work

### PR DESCRIPTION
**Pull request by gleocadie**
1. Little improvement for nuget
2. Removing Cast specifics (mainly namespace CastIL)
3. ProtozBuffer was not able to launch protoc.exe no protogen.exe (protocol buffer executables for Cpp/Java & C#)
   now when executing ProtozBuffer, the .proto file and high level classes are correctly generated in a custom namespace.

### CODE REVIEW

File: ProtoZBuffer/CommandLineExecuter.cs [hunk 1]
```diff
@@ -46,6 +46,11 @@ private static bool Execute(IOption option)
 
         static bool CommonVisit<T>(AbstractOption option) where T : AbstractGenerator, new()
         {
+            if (string.IsNullOrEmpty(option.Namespace) || string.IsNullOrWhiteSpace(option.Namespace))
```
Comment (christopheplat) [inline ProtoZBuffer/CommandLineExecuter.cs:49]: string.IsNullOrWhiteSpace also tests for empty strings, cf MSDN : Indicates whether a specified string is null, empty, or consists only of white-space characters.

File: ProtoZBuffer/ProtoZBuffer.nuspec [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>ProtoZBuffer</id>
+    <version>1.0.0.0</version>
+    <title>ProtoZBuffer</title>
+    <authors>CAST</authors>
+    <owners>CAST</owners>
+    <licenseUrl>http://LICENSE_URL_HERE_OR_DELETE_THIS_LINE</licenseUrl>
```
Comment (christopheplat) [inline ProtoZBuffer/ProtoZBuffer.nuspec:9]: We should probably write something here..

File: ProtoZBuffer/CommandLineExecuter.cs (hunk 1 above)
Comment (gleocadie) [inline ProtoZBuffer/CommandLineExecuter.cs:49]: Ok thank you.
I will fix it.

File: ProtoZBuffer/ProtoZBuffer.nuspec (hunk 2 above)
Comment (gleocadie) [inline ProtoZBuffer/ProtoZBuffer.nuspec:9]: We should complete the information. But I did not know under which licence ProtozBuffer was provided...
Still in progress :D
